### PR TITLE
Fix NZ Map projection and add a test for it.

### PR DESCRIPTION
**Pull request by scaddenp**
Fixes https://github.com/locationtech/proj4j/issues/62

### CODE REVIEW

**Comment by pomadchin** [comment]
Hi @scaddenp thank you so much for your contribution! 

To sign the commit you can run run `git commit --amend -s && git push origin -f master`.

To rebase and sign all the commits in this PR you can try the following:

```
git rebase --exec 'git commit --amend --no-edit -n -s' -i b4e02ab
git push --force
```

^ I didn't test it but in theory it should sign all the commits starting b4e02ab

**Comment by scaddenp** [comment]
Thanks but how do you execute a git command like that on github? I cant find a place to enter command line like that.

**Comment by pomadchin** [comment]
@scaddenp ah that is through the local git from your machine shell

**Comment by scaddenp** [comment]
And as I said, I do not have a local git. Installing anything on local machine is process with IT. It should not be this hard to contribute a bug fix.

File: src/main/java/org/locationtech/proj4j/proj/NewZealandMapGridProjection.java [hunk 1]
```diff
@@ -49,8 +49,8 @@
                  
     private final static double tpsi[] = { .6399175073, -.1358797613, .063294409, -.02526853, .0117879, -.0055161, .0026906, -.001333, .00067, -.00034 };
 
-    private final static double SECS_TO_RAD = ProjectionMath.DTR / 3600d;
-    private final static double RAD_TO_SECS = ProjectionMath.RTD * 3600d;
+    private final static double SECS_TO_RAD = 1E5 * ProjectionMath.DTR / 3600d;
+    private final static double RAD_TO_SECS = 1E-5 *ProjectionMath.RTD * 3600d;
```
Comment (pomadchin) [inline src/main/java/org/locationtech/proj4j/proj/NewZealandMapGridProjection.java:53]: Nice catch! [reference](https://github.com/OSGeo/PROJ/blob/46c47e9adf6376ae06afabe5d24a0016a05ced82/src/projections/nzmg.cpp#L38-L39)

File: src/main/java/org/locationtech/proj4j/proj/NewZealandMapGridProjection.java [hunk 2]
```diff
@@ -92,9 +92,9 @@ protected ProjCoordinate projectInverse(double x, double y, ProjCoordinate dst)
             dst.x = p.i;
             dst.y = tphi[tphi.length - 1];
             for (i = tphi.length - 1; i > 0; i--) {
-                dst.y = tphi[i] + p.r * dst.y;
+                dst.y = tphi[i-1] + p.r * dst.y;
```
Comment (pomadchin) [inline src/main/java/org/locationtech/proj4j/proj/NewZealandMapGridProjection.java:95]: :+1: [ref](https://github.com/OSGeo/PROJ/blob/46c47e9adf6376ae06afabe5d24a0016a05ced82/src/projections/nzmg.cpp#L71)

**Comment by pomadchin** [comment]
@scaddenp could you try to add the following text into your commit messages?

```
Signed-off-by: Name Surname <emailusedtoseigneca@mailservice.domain>
```

If that won't work I'll probably merge it as-is.

Thanks a lot for contributing and sorry for that overcomplicated process.

**Comment by scaddenp** [comment]
I tried adding 
Signed-off-by: Name Scadden <p.scadden@gns.cri.nz>
as comment to commit, but I could not find a way to edit the comment of an existing commit.
p.scadden@gns.cri.nz is the ECA email.

**Comment by pomadchin** [comment]
@scaddenp thanks a lot for your contribution!

**Pull request merged**